src/styles/blocks/stars__star.css

```
@@ -0,0 +1,27 @@
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--1 .stars__star:nth-child(-n + 1) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1), 
... ,
.stars--5 .stars__star:nth-child(-n + 5) {
```
Consider using commas to group selectors with the same styling